index.html

```diff
@@ -0,0 +1,191 @@
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/4.png" alt="Pizza foto">
+					<div class="pizza-title">SALIAMI</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/5.png" alt="Pizza foto">
+					<div class="pizza-title">SALSISHA</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/6.png" alt="Pizza foto">
+					<div class="pizza-title">WITH CHIKEN AND CHEADER</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/7.png" alt="Pizza foto">
+					<div class="pizza-title">PROSHOOTO</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/8.png" alt="Pizza foto">
+					<div class="pizza-title">MARGARITTA</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+			<a href="#">
+				<div class="block-pizza">
+					<img src="img/9.png" alt="Pizza foto">
+					<div class="pizza-title">CALSONE</div>
+					<div class="block-info">
+						<div>
+							<div class="timer-pizza">01:01:01</div>
+							<div class="ETA-pizza">ETA: 4 min</div>
+						</div>
+						<div>
+							<div class="id-pizza"></div>
+							<div class="price-pizza">20$</div>
+						</div>
+					</div>
+				</div>
+			</a>
+		</div>
+	</content>
+
+	<footer>
+		<address>
+			Kottans, Kottans Str. 1, tel 577-788-87
+		</address>
+		<div>
```

Comment:
The most fitting element for copyright information is the `<small>` element

Reply:
Where did you read this?

Reply:
> In HTML5, this element is repurposed to represent side-comments and small print, including copyright and legal text, independent of its styled presentation.

MDN

Reply:
Probably, you are right.

However, [spec notes](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-small-element) little bit another type of 'copyright'

Reply:
Actually, here is not copyright information. This is an _address_, so it should be an `<address>`.

In `<footer>`, of course.